src/app/modules/group/components/group-remove-button/group-remove-button.component.html

```diff
@@ -1,20 +1,20 @@
 <div class="container" *ngIf="state$ | async as state">
   <p-progressSpinner
       class="spinner"
-      *ngIf="state === 'loading' || (loading$ | async)"
+      *ngIf="state.isFetching || (deletionInProgress$ | async)"
       [style]="{width: '24px', height: '24px'}"
   >
   </p-progressSpinner>
-  <span class="has-children" *ngIf="state === 'hasChildren'" i18n>
+  <span class="has-children" *ngIf="state.data" i18n>
     Only an empty group can be deleted.
   </span>
-  <span class="error" *ngIf="state === 'error'" i18n>Error while loading the group info</span>
+  <span class="error" *ngIf="state.isError" i18n>Error while loading the group info</span>
   <button
       pButton
       type="button"
       class="p-button-rounded p-button-danger"
-      label="Delete this group"
+      i18n-label label="Delete this group"
       (click)="onDeleteGroup()"
-      [disabled]="state === 'hasChildren' || state === 'loading' || state === 'error' || (loading$ | async)"
+      [disabled]="state.data || state.isFetching || state.isError || (deletionInProgress$ | async)"
```

Comment:
the same here, I would prefer 
`[disabled]="(state.isReady && state.data) || state.isFetching || state.isError || (deletionInProgress$ | async)"

Reply:
Fixed